shopware/administration/6.6/bin/build-administration.sh

```diff
@@ -85,3 +85,4 @@ fi
 
 (cd "${ADMIN_ROOT}"/Resources/app/administration && npm run build)
 [[ ${SHOPWARE_SKIP_ASSET_COPY-""} ]] ||"${BIN_TOOL}" assets:install
+[[ ${SHOPWARE_SKIP_CACHE_CLEAR-""} ]] || "${BIN_TOOL}" cache:clear
```

Comment:
I think we should we should do this maybe only with an environment variable. I am not sure if we should delete always the cache after that commands

Reply:
ok but what against as a command option --keep-cache ? (The env variables are slowly becoming a little confusing to me)

Reply:
"--keep-cache" sounds like a good idea. 
If you build the administration (or the storefront) you do it with a good reason. And if you want to keep the cache - whyever - use the option.

Reply:
We can add the cache clearing for Storefront, as you need to clear the cache to see the changes. Admin is a static application and won't need a cleanup.

@BreakingTV do you want to work on that? Like adding it only to Storefront and adding a option?

Reply:
Added the code, should I add a help command, or should it just stay this way?